osu.Game/Storyboards/Drawables/DrawableStoryboard.cs

```diff
@@ -110,6 +99,16 @@ private void load(IGameplayClock? clock, CancellationToken? cancellationToken)
             }
 
             lastEventEndTime = Storyboard.LatestEventTime;
+
+            health = gameplayState?.HealthProcessor.Health.GetBoundCopy() ?? new BindableDouble(1);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            health.BindValueChanged(val => passing.Value = val.NewValue >= 0.5, true);
+            passing.BindValueChanged(_ => updateLayerVisibility(), true);
```

Comment:
Have you tested this with taiko, which doesn't use the draining health processor? It starts at 0 health and gradually increases over the duration of the map.

Reply:
See OP which already mentions that stable uses accuracy instead of health for displaying the layer.